src/cookbook/testing/integration/introduction.md

```diff
@@ -154,7 +154,7 @@ Now you can write tests. This involves three steps:
 import 'package:flutter_test/flutter_test.dart';
 import 'package:integration_test/integration_test.dart';
 
-import '../main.dart' as app;
+import 'package:counter_app/main.dart' as app;
```

Comment:
This is a snippet include. You'll need to modify the source referred to at the top of the block and run the snippet refresh tool referenced in the main README.

Reply:
The snippet has been updated, and the refresh tool now says 0 fragments need updating.